test/inductor/test_cutlass_backend.py

```diff
@@ -533,7 +535,7 @@ def mm(a, b):
             torch.testing.assert_close(Y_compiled, Y)
 
     # TODO: Enable dynamic test cases when dynamic support is added.
-    @unittest.skipIf(not SM80OrLater, "need sm_80")
+    @unittest.skipIf(not SM80, "need sm_80 exactly")
```

Comment:
Why does this need SM80 exactly, not SM80 or up?

Reply:
Hmm .. I think we should ensure that this isn't leading to exceptions or silently to incorrect results on SM90 systems. Do you know how it failed?